pkg/karmadactl/interpret.go

```diff
@@ -0,0 +1,142 @@
+package karmadactl
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/kubectl/pkg/util/templates"
+
+	"github.com/karmada-io/karmada/pkg/karmadactl/util"
+)
+
+var (
+	interpretLong = templates.LongDesc(`
+		Check the valid of customizations. Or show the result of the rules running.
```

Comment:
It seems not easy to understand, what is `customizations`, is it designed for `interpreter`?

Reply:
@RainbowMango add a new API named `ResourceInterpreterCustomization`
https://github.com/karmada-io/karmada/blob/c669017ce344332dc6da55495e25c28bde861c7e/pkg/apis/config/v1alpha1/resourceinterpretercustomization_types.go#L17

Reply:
Got it, suggest attach `interpreter`, like `Check the valid of interpreter customizations`